### PR TITLE
navbar changes v1 #1338

### LINKED ISSUE #1338: Navbar Updates
### Overview
We need to modify some of the changes made to the navbar.

### Action Items
Make the following changes in `components/Navbar.js` file:
- [x] Change `1rem` to `20px`
- [x] Padding under the active link and link title.  12px
- [x] Change mt in Box from `4` to `2`

### PR DESCRIPTION
**Pull request by plang-psm**
Fixes #1338

### What changes did you make and why did you make them ?

  - Change the font size from 1rem to 20px.
  - Changed the padding of the active underline by adding 10px like the Figma.
       - See the comments for possible additional change. 
  - Updated the mt from 4 to 2 to match Figma margin.

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals before changes are applied</summary>

image

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
image

</details>

### CODE REVIEW

**Comment by plang-psm** [comment]
The current Figma design has a padding of 10px between the Nav link and the underline active style. I noticed that the styled button has some space that I wasn't able to remove. Open to suggestions but setting margin/padding to 0 did not work so I played with the px and found that setting it to 6px matched the Figma better. I submitted with 10px but can switch to 6px if everyone agrees.

<details>
<summary>Spacing on the nav link compared to figma</summary>
<br>
image
- 10px padding on site

image
- 10px padding on Figma

</details>

<details>
<summary>6px padding</summary>
<br>

image
- 6px padding

image
- Figma padding

image
- 10px padding

</details>

**Comment by MattPereira** [comment]
@plang-psm  i think the mui Buttons come with a default minimum height but its not worth the effort of figuring out because your `6px` padding fix is more than sufficient for this use case and matches the figma perfectly

File: client/src/components/Navbar.js [hunk 1]
```diff
@@ -12,9 +12,9 @@ const Navbar = (props) => {
  
   const StyledButton = styled(Button)({
     color: '#757575',
-    padding: '0rem 0rem',
+    padding: '10px 0',
```
Comment (MattPereira) [inline client/src/components/Navbar.js]: i agree with you that `6px` looks better

**Review by MattPereira — CHANGES_REQUESTED** [review]

File: client/src/components/Navbar.js (hunk 1 above)
Comment (plang-psm) [reply client/src/components/Navbar.js]: Font size has been updated!

**Review by MattPereira — APPROVED** [review]

**Pull request merged**